libs/libskynet/.prettierignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
You should also add package.json to that list, otherwise any time that dependabot adds new dependency package.json file will get totally rewritten because of the tab indentation.